config/jobs/kubernetes-sigs/testing_frameworks/testing-frameworks-config.yaml

```diff
@@ -7,13 +7,35 @@ presubmits:
     skip_report: false
     labels:
       preset-service-account: "true"
+      preset-dind-enabled: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+      - image: gcr.io/k8s-testimages/bootstrap:latest
```

Comment:
Can you use pod-utilities (decorate: true) instead of bootstrap?

Here's an example job which does this:
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/org/kubernetes-org-jobs.yaml